back_end/bug_app/serializers.py

```diff
@@ -13,18 +13,18 @@ class SprintSerializer(ModelSerializer):
     tickets = TicketSerializer(many=True)
 
     class Meta:
-        model: Sprint
```

Comment:
This bug was the one that caused my Sprint api calls to fail. I thought the problem was in the database, which caused a whole other mess but it here. It took me 10 hours to get one step in front of where I was when I woke up this morning.